examples/btc-to-evm-fungible-transfer/src/utils.ts

```diff
@@ -0,0 +1,32 @@
+export async function getFeeEstimates(blockstreamUrl: string): Promise<number> {
```

Comment:
I would have these functions under `blockstream-api.ts` or similar file name

Reply:
instead of utils file?

Reply:
yes